### ISSUE TITLE
Automated docker build failing

### ISSUE THREAD

**Issue by adamlc** [djenriquez/nomad#11]
The automated docker build is currently failing for 0.7.1 :(

**Comment by james-powis** [comment]
Not sure how to address, but can we get 0.7.1 cooked and up, it contains a feature I so direly need...

**Comment by djenriquez** [comment]
Is it the `sysctl` stanza @james-powis? 😉  I'll take a quick look tonight why its failing, seems to be intermittent and the `docker build` works fine locally. For now I'll manually push up 0.7.1.

**Comment by james-powis** [comment]
I was having numerous issues pulling the gosu gpg signing key earlier today when trying to bake it myself... Glad to see this got pushed, thanks much!

**Comment by adamlc** [comment]
Yeah looks like the gpg keyserver is down or having issues, thanks for manually pushing the build :)
